### DISCUSSION TITLE
TypeError: Object.groupBy is not a function

### DISCUSSION THREAD

**Discussion by jim-king-2000** [vercel/community#5874] [Feature Question] [answered]
### Question

Set the node.js vertion to "20.x" and still get this error message. See <https://vercel.com/cache-hit/eda-frontend/6kJETPeCgeR7Hb1G1bhHzAeegSZQ> for the detail.

I notice that `Object.groupBy()` was introduced in Node.js since version 21.0.0. So, does it mean that the nodejs version vercel is using are still too old to support `Object.groupBy()`?

**Comment by swarnava** [comment, marked as answer]
Hi,

We are using v20.11.0 at this moment in build container.

**Comment by jim-king-2000** [comment]
I strongly suggest that we should pump it up to v21.6. Now I have to fallback to lodash, I don't like lodash.
